src/functions-templates/javascript/edge/hello/{{name}}.js

```diff
@@ -0,0 +1,4 @@
+export default async (Request) =>
+  new Response('Hello, World!', {
+    headers: { 'content-type': 'text/html' },
+  })
```

Comment:
I'm not sure we want to use arrow functions in the templates, especially given the associated examples on the [edge functions example site](https://edge-functions-examples.netlify.app/example/hello) doesn't use arrow functions. Plus, I know a lot of people that this syntax still confuses.

I would recommend 

```
export default async (Request) => {
  return new Response("Hello, World!", {
    headers: { "content-type": "text/html" },
  });
};
```

Reply:
If I may piggyback on this, I don't think we should be using `Request` (with a capital R) for the function argument. By convention, that is typically used for classes (like `Response`), but this argument is an instance of a class (and the class is `Request`).

I think this can generate some confusion, especially if you want to create another request inside of the function, because you've overridden the `Request` identifier, so doing `const req = new Request(...)` will throw.

Reply:
Also, I'm a bit confused about the arrow function question, because your recommendation is also using an arrow function. Are you talking about not using an implicit return?

Reply:
You're definitely right about `Request`.

> Are you talking about not using an implicit return?

Sorry, yes, exactly. (Sorry I had very little sleep last night!)

Reply:
Okay, I believe I added explicit return statements to all the templates now, although they still use arrow functions ;) At any rate, they match the functions on the example site!